More examples (#11)
DevExpress/testcafe-examples: examples/wait-for-file-download/README.md

# Wait For File Download

**Test Code**: [test.js](test.js)

> This example shows how to set a test to wait for the file download.

Reviewer: ```suggestion
This example shows how to specify that a test should to wait for the file to download.
```

> Before the test begins, it obtains the `downloadedFilePath`. If the file already exists on the path, the test removes the file to save the download without warnings or errors.

Reviewer: ```suggestion
Before the test begins, it obtains the `downloadedFilePath`. If the file already exists, the test removes the file and saves the downloaded file without warnings or errors.
```

> In the example, the test runs on Google Chrome browser. The asynchronous `waitForDownload` function runs a `for` loop which checks if the file exists on the specified download path. It runs the body of the loop several times, with a timeout between tries, until the file is downloaded. If the function does not locate the file after the tries, it returns `false`.

Reviewer: "exists on the specified download path" Why 'on' the path?

Reviewer: ```suggestion
In the example, the test runs on Google Chrome browser. The asynchronous `waitForDownload` function runs a `for` loop which checks if the file exists in the specified download path. The body of the loop is run several times with a timeout between attempts until the file is downloaded. If the function cannot locate the file, it returns `false`.
```
